gpiozero/boards.py

```diff
@@ -1185,3 +1185,86 @@ def on(self):
     def off(self):
         self.value = False
 
+class SevenSegmentDisplay(LEDBoard):
+    layouts = {
```

Comment:
If the layout contemplate that both possibilities, I am in favor of a way to get (@property only) the string representation of what is being displayed: char + dp